float32 velocity
float32 t
float32 pitch
string target_model_name
float32[] target_position